Formula/leela-zero.rb

```diff
@@ -0,0 +1,37 @@
+class LeelaZero < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://zero.sjeng.org/"
+  url "https://github.com/gcp/leela-zero/archive/v0.15.tar.gz"
+  sha256 "6a34df356cf64aaa74bdf90826f8653255c42c3e65da8715f9c39751de00926a"
+
+  option "without-network", "Don't automatically download the best known network"
+
+  depends_on "boost"
+
+  resource "network" do
```

Comment:
Unless this resource is versioned, like the software, it's probably best not to ship it.

Reply:
It is versioned (by hash), but it changes relatively frequently (every few days, far more often than we'd want to upgrade this formula). The `/best-network` endpoint that I'm using is a shortcut that says "get the currently best one", but for obvious reasons there's no fixed hash.

So if you prefer, I can just pick a fixed network and just let it get out of date. Not shipping with a network would be pretty useless, you can't use leela-zero at all without weights.

Is that preferable to just having an unversioned resource?